python/hail/expr/types.py

```diff
@@ -154,6 +180,9 @@ class TInt32(Type):
     In Python, these are represented as :obj:`int`.
     """
 
+    _min_value = -(1 << 31)
+    _max_value = (1 << 31) - 1
```

Comment:
Not sure why these are here, can just return in min_value, etc?

Reply:
yeah, true.